plutus-core-spec/plutus-core-specification.tex

```diff
@@ -828,14 +830,17 @@ \subsection{Term reduction}
 Execution of Plutus Core programs is performed by (possibly
 non-terminating) reduction of well-typed terms.  The reduction rules
 are contained in Figure~\ref{fig:term-reduction}, and give us a fairly
-standard operational semantics.  
+standard operational semantics.
 
 \subfile{figures/TermReduction}
 
+\todokwxm{James says that the rule for \texttt{abs} breaks type preservation.
+  See issue \#1145.  This is also related to type erasure.}
```

Comment:
Should be an easy fix: just do the type substitution in the abs rule. Can we just do it?